docs/software/advanced_networking.md

```diff
@@ -255,13 +255,18 @@ It is best suited for high traffic web sites and powers quite a number of the wo
     systemctl start haproxy    
     ```
 
+=== "Access to the web interface"
+
     The stats web interface is accessible via port **1338**:
 
     - URL = `http://<your.IP>:1338`
     - Prometheus metrics = `http://<your.IP>:1338/metrics`
     - Username = `admin`
     - Password = `<yourGlobalSoftwarePassword>` (default: `dietpi`)
 
+    [//]: # (Include Avahi Daemon <hostname>.local access textblock)
+    --8<---------- "snippet-includes/AvahiDaemon-WebInterface-access_textblock.md"
```

Comment:
The syntax, snippets dir and file name make the comment above sort of redundant, what you think?

I think we can generally skip them by just using meaningful file names. And `snipped-includes` says what the line does.

Reply:
No need to remove them here, just an idea for the future. If you prefer the additional comment for visibility, no problem by me to put them in.